### ISSUE TITLE
Presence Stops

### ISSUE THREAD

**Issue by PouriDev** [hugolgst/vimsence#31]
When I start application two or three times, or switch between files, Vim sense stops, I'm using my discord application

**Comment by anned20** [comment]
@MrNilDev, sorry that it took a while before getting to this issue.

Are you still experiencing this issue? I can't reproduce it.

**Comment by MrNilDev** [comment]
Yes, It still happens. Like it hungs or looking for something, or maybe my internet connection is so low that it can't do the needed jobs. Also it really makes `Neovim` or `vim` slow so I decided to delete it for now

**Comment by ColexDev** [comment]
I am having the same exact issue. It makes Neovim take about 2-3 seconds to start working and when you switch between files it does not work. When you have multiple instances of neovim open the plugins seems to stop working all together. It also seems to not work on some files sometimes.

**Comment by JuniorSuperTux** [comment]
That issue occurs me on vim as well. Opening vim becomes slower and the presence disappears after switching between files. However, switching tabs does not affect presence.
